clients/spark-extensions-basetests/src/main/java/org/projectnessie/spark/extensions/AbstractNessieSparkSqlExtensionTest.java

```diff
@@ -374,7 +374,7 @@ void useShowReferencesAtTimestampWithTimeZone(DateTimeFormatter dateTimeFormatte
     assertThat(sql("USE REFERENCE %s AT `%s` IN nessie ", refName, timeWithZone))
         .containsExactly(row("Branch", refName, lastCommitBeforeTimePredicate.getHash()));
     assertThat(sql("SHOW REFERENCE IN nessie"))
-        .containsExactly(row("Branch", refName, lastRefHashGlobally));
+        .containsExactly(row("Branch", refName, lastCommitBeforeTimePredicate.getHash()));
```

Comment:
Not sure why this change is necessary, because the logic for timestamps shouldn't change, right?
And there's no more check against the reason mentioned in the comment - and the variable is now unused.

Reply:
> Not sure why this change is necessary, because the logic for timestamps shouldn't change, right?

`USE ref AT hash or timestamp` has the same behaviour (just that timestamp will be mapped to a hash -> uses `NessieUtils#calculateRef`). 

Without this PR, after setting `USE reference at oldHashTimestamp`, show reference was always giving the head commit hash of that reference. Hence, need to modify it now as the show reference will return the current hash under use.

> and the variable is now unused.

Oh, I will remove it.